src/Exporter.php

```diff
@@ -18,6 +18,10 @@ class Exporter implements ExporterInterface
 </graphml>
 EOL;
 
+    /**
+     * @param \Fhaculty\Graph\Graph $graph
```

Comment:
I don't think there's any "popular" convention. Symfony appears to follow the short names, while Laravel prefers the FQN. I'll change it to short names if that is what you prefer.